libs/runtime/ebpf_platform.c

```diff
@@ -43,25 +43,23 @@ _Requires_lock_not_held_(*lock) _Acquires_lock_(*lock) _IRQL_requires_max_(DISPA
 ebpf_lock_state_t
 ebpf_lock_lock(_Inout_ ebpf_lock_t* lock)
 {
-    return KeAcquireSpinLockRaiseToDpc(lock);
-}
+    KIRQL old_irql = KeGetCurrentIrql();
```

Comment:
I am approving but this is another case where Ke*() calls should be in ebpf_platform_kernel.c rather than ebpf_platform.c.  Alternatively, ebpf_platform.c should have cxplat_*() calls.  But it was this way already so fine to leave for future cleanup.

Reply:
Done. (We should consider removing the entire ebpf_lock_t abstraction at some point).

Reply:
Rolled back for now.